### PR TITLE
Better Powershell install

### PR DESCRIPTION
**Pull request by admiralAwkbar**
Making PowerShell install faster, cleaner.. better...

### CODE REVIEW

**Comment by admiralAwkbar** [comment]
@zkoppert on my local build it was about 2 min faster... so it should help
seems ARM ttk is where the next chunk is at...

File: Dockerfile [hunk 1]
```diff
@@ -96,21 +93,20 @@ ENV PATH="/node_modules/.bin:${PATH}"
...
-    && curl -s https://api.github.com/repos/powershell/powershell/releases/${PWSH_VERSION} \
-    | grep browser_download_url \
-    | grep linux-alpine-x64 \
-    | cut -d '"' -f 4 \
-    | xargs -n 1 wget -O - \
-    | tar -xzC ${PWSH_DIRECTORY} \
-    && ln -sf ${PWSH_DIRECTORY}/pwsh /usr/bin/pwsh \
-    && pwsh -c 'Install-Module -Name PSScriptAnalyzer -RequiredVersion ${PSSA_VERSION} -Scope AllUsers -Force'
+COPY --from=powershell /usr/bin/pwsh /usr/bin/
+CMD /usr/bin/pwsh -c 'Install-Module -Name PSScriptAnalyzer -Scope AllUsers -Force'
```
Comment (nemchik) [inline Dockerfile:109]: This should be `RUN` (not `CMD`)

**Comment by admiralAwkbar** [comment]
@nemchik when set to `RUN` and not `CMD` it fails to see the bin dir object we just set... this is driving me a bit nuts lol

**Comment by nemchik** [comment]
> @nemchik when set to `RUN` and not `CMD` it fails to see the bin dir object we just set... this is driving me a bit nuts lol

@admiralAwkbar
- `CMD` executes inside the container (after it is built). Ex: The default command to be run when an end user runs `docker run github/super-linter` would be whatever is in `CMD`.
-  `RUN` executes while building the container. Ex: I need X, Y, or Z installed inside the container so that when it runs it will have those things.

Ref:
https://docs.docker.com/engine/reference/builder/#cmd
https://docs.docker.com/engine/reference/builder/#run

The big question here is whether the `pwsh` binary executable is self contained and portable or if it requires everything else found in the tar.gz that was originally used to install. If it does require all the other stuff we might not be able to pull only the one file (might be best to keep the current method).

**Comment by admiralAwkbar** [comment]
So too many deps not fulfilled.... may have to wait until the better image is available... still fun to try :)

**Pull request closed without merging**